### PR TITLE
Build frontend

### PR DESCRIPTION
**Pull request by isabellaaston**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<head>
```
Comment (HiredMussel) [inline index.php]: You'll still need an html tag to start the document with

File: index.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+<link rel="stylesheet" type="text/css" href="normalize.css">
+<link rel="stylesheet" type="text/css" href="style.css">
+<link href="https://fonts.googleapis.com/css?family=Fjalla+One&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css?family=Questrial&display=swap" rel="stylesheet">
```
Comment (HiredMussel) [inline index.php]: No indentation here. You might consider indenting the contents of your head tag to make your code more readable

File: index.php [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    <h1>My Handbag Collection</h1>
+    <nav>
+        <div class="navbar">
+            <a class="underline">Collection</a>
+            <a class="underline">Add Item</a>
+            <a class="underline">Edit Item</a>
+        </div>
+    </nav>
+<main>
+</main>
```
Comment (HiredMussel) [inline index.php]: Empty main tag? If it isn't doing anything then delete it.

File: index.php [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+</body>
+<?php
+$db = new PDO('mysql:host=db; dbname=MyCollection','root','password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```
Comment (HiredMussel) [inline index.php]: You might want to move the two lines above to a function - it was recommended by Ash during my code review and it's likely that you'll have to connect to a database again at some point during the next story

File: index.php [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+$query=$db->prepare("SELECT* FROM `bags`;");
```
Comment (HiredMussel) [inline index.php]: Syntax is SELECT * FROM `bags`;
You are missing a space. This probably doesn't matter for functionality (as MySQL probably deals with it in interpretation) but I'd still amend it.

Also, recall that it is bad practice to use 'SELECT *' at all, as it is possible that your database structure will change during development. I would recommend that you SELECT only the values you want.

File: index.php [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+    echo '<br>';
+    echo '<li>';
+    echo '£'.$bag['price'];
+    echo '</li>';
+    echo '<br>';
+    echo '<li>';
+    echo $bag['materials'];
+    echo '</li>';
+    echo '</ul>';
+    echo '</div>';
```
Comment (HiredMussel) [inline index.php]: While functionally valid, the above is quite hard to read. I would recommend that you try and lay out the code above in a way which resembles the HTML it is generating more closely.

File: style.css [hunk 7]
```diff
@@ -0,0 +1,53 @@
+* {
+	box-sizing: border-box;
+}
+body {
+    background-color: #FFFFFF;
+    margin: 0;
+    padding: 0 10% 0 10%;
```
Comment (HiredMussel) [inline style.css]: Is this necessary? Try padding: 0 10% and see if it works.

File: style.css [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+.image {
+    display: flex;
+    justify-content: center;
+} 
+img {
+    width: 90%;
+}
+.bags {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
```
Comment (HiredMussel) [inline style.css:38]: Using CSS Grid? Nice

Comment (isabellaaston) [reply style.css:38]: thanks! Rich helped me out, and i really like it! need to read up on it!

**Pull request merged**